components/search/SearchForm.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <b-form
+    inline
+    @submit.prevent="$emit('submit:searchForm')"
+  >
+    <b-form-input
+      ref="query"
```

Comment:
This ref can be deleted.

Reply:
Indeed, and it has been in the following commit.